code/modules/mob/living/simple_animal/hostile/wolf.dm

```diff
@@ -319,8 +317,8 @@
 			if(0 to -INFINITY)
 				to_chat(user, "<span class='warning'>It seems, well, dead.</span>")
 		switch(alpha_stance)
-			if(ALPHANONE)
-				//Nuthin
+			//if(ALPHANONE)
+				//Nuthin, was gonna have barking, but PJB got angry
```

Comment:
Hey now if you want to add barking go ahead, I was just pointing out that having an empty case for a switch is silly.